system-tests/common.py

```diff
@@ -316,28 +309,27 @@ def tear_down(self):
             shutil.rmtree(self._repo_path)
 
 
-class CheckTask(systest.TestCase, metaclass=ABCMeta):
+class CheckTask(ABC, metaclass=ABCMeta):
```

Comment:
Inheriting `ABC` implies a metalcass of `ABCmeta`.

Reply:
Ah yes, of course it does!